javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -91,4 +111,8 @@ private static void runAndReport(Class<?> benchmarkClass, int warmupIterations,
         }
         return results;
     }
+
+    public static int xor(int x, int y) {
```

Comment:
I could rename it to `aggregate`, if that would help.
It's only used as validity check for `iteration` and `get` in the benchmarks to avoid dead code elimination.